tools/utils.cc

```diff
@@ -114,11 +125,21 @@ int tool_app_template::run_async(int argc, char** argv, noncopyable_function<int
         }
     }
 
+    auto init = app.get_options_description().add_options();
+    configurable::append_all(*_dbcfg, init);
```

Comment:
Can we make this optional? Currently scylla-sstable is the only tool that nneds this. We can add a flag to `tool_app_template::config` that only scylla-sstable would set. Then we would enclose all this new code in `if`. Ugly, I know, but I would like to keep these options out of the CMD UI of e.g. nodetool.

Reply:
Sure. Pushed change to make it fully config-local, so it only touches sstables tool.